### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by Thealexbarney**
This PR's split into multiple commits for easy viewing and removal of each change, as I anticipate some of the changes will be reverted

### CODE REVIEW

**Comment by jduncanator** [comment]
Please don't change `var` to things like `DecoderHelper.BitMask`. I tend to use `var` more liberally than others would like, but this sort of change is crazy, and a perfect example for a reasonable use of `var`.

File: ChocolArm64/Translation/TranslatedSub.cs [hunk 1]
```diff
@@ -75,16 +75,16 @@ public long Execute(CpuThreadState threadState, MemoryManager memory)
...
-                return false;
-            }
+           if (_callCount++ < MinCallCountForOpt)
+           {
+               return false;
+           }
 
-            //Only return true once, so that it is added to the queue only once.
+           //Only return true once, so that it is added to the queue only once.
             _rejit = false;
```
Comment (jduncanator) [inline ChocolArm64/Translation/TranslatedSub.cs]: Why has the spacing changed? Either `_rejit` is mis-aligned, or the comment should still have the space at the start.

**Comment by jduncanator** [comment]
Please don't change files in the Native folder of libsoundio, these files are automatically generated using a tool and will change back whenever they're updated.

File: ChocolArm64/Translation/TranslatedSub.cs (hunk 1 above)
Comment (Xpl0itR) [reply ChocolArm64/Translation/TranslatedSub.cs]: it looks like `_rejit` is misaligned, the `return` statement is too i think

Comment (gdkchan) [reply ChocolArm64/Translation/TranslatedSub.cs]: I think the right fix here is adding one extra space before `if (!_rejit)`.

File: Ryujinx.Graphics/Gal/GalImageFormat.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿// ReSharper disable InconsistentNaming
```
[the remark this answers is not in the document]
Comment (Thealexbarney) [reply Ryujinx.Graphics/Gal/GalImageFormat.cs]: The file is full of enums values that don't always match the standard naming convention. That line suppresses those warnings in ReSharper

**Comment by Thealexbarney** [comment]
Some of the changes were removed and comment spacing fixes were added.

**Comment by Thog** [comment]
Need a rebase and it's ready to go 👍

**Comment by Thealexbarney** [comment]
It should be good to go

File: Ryujinx.HLE/HOS/Services/Am/IHomeMenuFunctions.cs [hunk 3]
```diff
@@ -23,7 +23,7 @@ public IHomeMenuFunctions(Horizon system)
                 { 21, GetPopFromGeneralChannelEvent }
             };
 
-            //ToDo: Signal this Event somewhere in future.
+            // ToDo: Signal this Event somewhere in future.
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Am/IHomeMenuFunctions.cs]:
```suggestion
            // TODO: Signal this Event somewhere in future.
```

File: Ryujinx.HLE/HOS/Services/Friend/IFriendService.cs [hunk 4]
```diff
@@ -108,7 +108,7 @@ public long UpdateUserPresence(ServiceCtx context)
             long position = context.Request.PtrBuff[0].Position;
             long size     = context.Request.PtrBuff[0].Size;
 
-            //Todo: Write the buffer content.
+            // Todo: Write the buffer content.
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Friend/IFriendService.cs]:
```suggestion
            // TODO: Write the buffer content.
```

File: Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs [hunk 5]
```diff
@@ -1014,7 +1014,7 @@ public long SendVibrationValues(ServiceCtx context)
                 context.Request.PtrBuff[1].Position,
                 context.Request.PtrBuff[1].Size);
 
-            //Todo: Read all handles and values from buffer.
+            // Todo: Read all handles and values from buffer.
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs]:
```suggestion
            // TODO: Read all handles and values from buffer.
```

File: Ryujinx.Tests.Unicorn/IndexedProperty.cs [hunk 6]
```diff
@@ -9,8 +9,8 @@ public class IndexedProperty<TIndex, TValue>
 
         public IndexedProperty(Func<TIndex, TValue> getFunc, Action<TIndex, TValue> setAction)
         {
-            this.GetFunc = getFunc;
-            this.SetAction = setAction;
+            GetFunc = getFunc;
```
Comment (AcK77) [inline Ryujinx.Tests.Unicorn/IndexedProperty.cs]: Align assignments

File: Ryujinx.Audio/Renderers/SoundIo/SoundIoAudioOut.cs [hunk 7]
```diff
@@ -206,7 +206,7 @@ private static SoundIODevice FindNonRawDefaultAudioDevice(SoundIO audioContext,
                 return defaultAudioDevice;
             }
 
-            for(var i = 0; i < audioContext.BackendCount; i++)
+            for(int i = 0; i < audioContext.BackendCount; i++)
```
[the remark this answers is not in the document]
Comment (Thealexbarney) [reply Ryujinx.Audio/Renderers/SoundIo/SoundIoAudioOut.cs]: Fixed the spacing on all the `for`s and `if`s

**Comment by AcK77** [comment]
Thanks for this massive cleanup !

**Pull request merged**